utils/src/activityLog/activities/ExecuteActivity.ts

```diff
@@ -24,6 +24,12 @@ export class ExecuteActivity<TContext extends types.ExecuteActivityContext = typ
         }
     }
 
+    protected override report(progress?: { message?: string; increment?: number }): void {
+        const message: string | undefined = this.context.activityChildren?.length ? this.timerMessage : progress?.message;
```

Comment:
nit: I think a comment might be worthwhile here rationalizing why we decided to keep progress.message when there's no activity children.